pkg/queue/stats.go

```diff
@@ -88,11 +98,17 @@ func NewStats(podName string, channels Channels, startedAt time.Time) *Stats {
 				updateState(event.Time)
 
 				switch event.EventType {
+				case ProxiedIn:
+					proxiedCon++
+					fallthrough
```

Comment:
Why do we need `fallthrough` here? The event will be only one of these cases.

Reply:
I want to keep concurrency's meaning of requests that are pending, no matter where the requests are from.